src/app/routes/api/v3/corppass/corppass.routes.ts

```diff
@@ -14,10 +14,11 @@ export const CorppassOidcRouter = Router()
  * @route GET api/v3/corppass/.well-known/jwks.json
  * @returns 200
  */
-CorppassOidcRouter.use(
-  '/.well-known/jwks.json',
-  express.static(spcpMyInfoConfig.cpOidcRpJwksPublicPath),
-)
+CorppassOidcRouter.get('/.well-known/jwks.json', (req, res) => {
+  const jwksContent = spcpMyInfoConfig.cpOidcRpJwksPublic
```

Comment:
The res.json() function sends a JSON response. This method sends a response (with the correct content-type) 

can use this